### PR TITLE
⚗️ Code-review comments/reflections

### PR DESCRIPTION
**Pull request by s373r**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -3,8 +3,9 @@
...
   "semi": false,
   "singleQuote": true,
+  // NOTE(DP): tabs nooooooo!
   "useTabs": true,
   "jsxSingleQuote": true,
   "bracketSpacing": true,
   "arrowParens": "avoid"
-}
\ No newline at end of file
+}
```
Comment (s373r) [inline .prettierrc:11]: Do not forget for last blank lines

File: package-lock.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "monsters-of-chopa-sipa",
+  "name": "ts-tg-bot",
   "version": "1.0.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "name": "monsters-of-chopa-sipa",
+      "name": "ts-tg-bot",
```
Comment (s373r) [inline package-lock.json:8]: It changed by some reason

Q: Is that related to some GH template kung fu?

File: package.json [hunk 3]
```diff
@@ -5,13 +5,13 @@
   "main": "build/index.js",
   "scripts": {
     "compile": "tsc",
-    "prepare": "npm run compile && npm run copyFiles",
-    "bg:start": "npm run prepare && pm2 start build/index.js --name telegram-bot",
-    "bg:restart": "npm run prepare && pm2 restart telegram-bot",
+    "postcompile": "cp -r ./src/locales/ ./build/",
+    "bg:start": "npm run compile && pm2 start build/index.js --name telegram-bot",
+    "bg:restart": "npm run compile && pm2 restart telegram-bot",
```
Comment (s373r) [inline package.json:10]: Several optional comments
1) [pre/post scripts](https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts) are your friends if you like to do some... pre/post actions :) And if you don't want to figure out new names (`copyFiles` is too general for me)
2) `prepare` is called every time when `npm i`. I dunno is it intentional or not -- just FYI

File: package.json [hunk 4]
```diff
@@ -5,13 +5,13 @@
...
     "bg:logs": "pm2 logs telegram-bot --lines 1000 -f",
-    "start": "npm run prepare && node .",
-    "pretty": "npx prettier --write src",
-    "copyFiles": "cp -r src/locales build"
+    "prestart": "npm run compile",
+    "start": "node .",
+    "lint": "npx prettier --write src"
```
Comment (s373r) [inline package.json:14]: `lint` (or might be better `format`) more expected naming

**Comment by s373r** [comment]
@ExposedCat , please close PR when you check it
(it is shown in my active PRs, and I'd like to hide it :smile: )

File: package-lock.json (hunk 2 above)
Comment (ExposedCat) [reply package-lock.json:8]: @s373r nope, it's because I was making a template from a new project :)

File: src/config/app.ts [hunk 5]
```diff
@@ -1,8 +1,13 @@
+// NOTE(DP): use instead of ../../-like imports path-aliases
+//           see more:
+//           https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353
 import { loadEnv } from '../helpers/load-env.js'
 import { validateEnv } from '../helpers/validate-env.js'
 import { startBot } from './bot.js'
 import { connectToDb } from './database.js'
 
+// NOTE(DP): looks like startApp() is not related to src/config/ scope
```
Comment (ExposedCat) [inline src/config/app.ts:9]: Initialization of app, the same concept as the rest of files there. Probably `config` should be renamed

File: src/config/bot.ts [hunk 6]
```diff
@@ -59,7 +61,11 @@ export async function startBot(database: Database) {
 	const bot = new TelegramBot<CustomContext>(process.env.TOKEN)
 	extendContext(bot, database)
 	setupMiddlewares(bot, i18n)
+	// NOTE(DP): I guess we need to log users actions, e.g. start, stop, etc
```
Comment (ExposedCat) [inline src/config/bot.ts:64]: Does not look useful / will be a lot of entries

**Comment by ExposedCat** [comment]
Fixed in #2

File: src/config/bot.ts (hunk 6 above)
Comment (s373r) [reply src/config/bot.ts:64]: @ExposedCat Ideally, general recommendation to treat logs as events… there is a good article w/ explorations, but I cannot find it quickly :(

anyway, in this case we don't create extra logs only for useful for debugging things
